beard/similarity/pairs.py

```diff
@@ -229,6 +231,113 @@ def transform(self, X):
         return Xt.reshape((n_samples, 1))
 
 
+class EstimatedFeature(TransformerMixin):
```

Comment:
Can you rebrand this as `EstimatorTransformer` and rename `classifier` to `estimator` everywhere in the class?